src/schemas/Program/index.js

```diff
@@ -189,7 +189,7 @@ const typeDefs = gql`
 		"""
 		retrieve all DataCenters
 		"""
-		dataCenters: [DataCenter]
+		dataCenters(shortName: String): [DataCenter]
 	}
```

Comment:
add optional parameter in the dataCenter query. The parameter is the shortName of a program.